horovod/tensorflow/__init__.py

```diff
@@ -237,3 +248,84 @@ def get_slot_names(self, *args, **kwargs):
     def variables(self, *args, **kwargs):
         """Calls this same method on the underlying optimizer."""
         return self._optimizer.variables(*args, **kwargs)
+
+
+class _DistributedGradientTape(tf.GradientTape):
+    """An tape that wraps another tf.GradientTape, using an allreduce to
```

Comment:
Can you move this description to the `def DistributedGradientTape`?

Reply:
ok. i move it to ```DistributedGradientTape```